### DISCUSSION TITLE
Change base map style

### DISCUSSION THREAD

**Discussion by manuelgamelas** [bozdoz/wp-plugin-leaflet-map#241] [Q&A]
Hello everyone. 
I need a different base maps style for my map how can i do that, something like image
![imagem](https://github.com/bozdoz/wp-plugin-leaflet-map/assets/88498249/40ac6692-35cf-4abb-a9a1-baf7fd5749b9)

**Comment by bozdoz** [comment]
Looks like you need to specify a new tile url. Check the readme or the shortcode example page for examples

**Comment by manuelgamelas** [comment]
In the examples I didn't find how to put the map style in the format I want. Maybe I'm not looking well enough. Could they help?

**Reply by bozdoz** [reply]
If you're using map box they have kind of infinitely customizable tiles. Otherwise you'd have to find the specific tiles you want. Then in this plug-in use the appropriate url for the "src" attribute.
